File: zwkfb_view/src/main/java/安卓/视图/布局解析器.java
package 安卓.视图;

import android.content.Context;
import android.view.LayoutInflater;

/**
 * 创建时间：2025年11月26日.
 * <p>
 * 描述：布局解析器
 * @author dxyc
 */
public abstract class 布局解析器 extends LayoutInflater {

    protected 布局解析器(Context 上下文) {
        super(上下文);
    }

    protected 布局解析器(LayoutInflater 原始的, Context 新上下文) {
        super(原始的, 新上下文);
    }


}

File: zwkfb_view/src/main/java/安卓/视图/布局解析器.kt
package 安卓.视图

import android.content.Context
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup


//======================================================================

/**
 * 描述：解析器
 * @param 布局id 布局资源id
 * @param 根布局 根布局
 * @param 附加到根布局 是否附加到根布局
 * @return 解析后的视图
 */
fun LayoutInflater.解析器(布局id: Int, 根布局: ViewGroup?, 附加到根布局: Boolean): View =
    inflate(布局id, 根布局, 附加到根布局)


